src/content/ui-components/getting-started/style.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Style Tiptap UI Components
+meta:
+  title: UI Components | Tiptap
+  description: Style Tiptap UI Components to match your design system or application theme.
+  category: UI Components
+---
+
+import { Callout } from '@/components/ui/Callout'
+
+### Add Styles
+
+To enable support for SCSS, install the appropriate SCSS compiler based on your framework setup.
+
+<Callout title="Styles are only added when using the CLI" variant="info">
+  Styles are included only if you installed a component or template using the CLI. These styles come
+  bundled with whatever you add through the CLI.{' '}
+</Callout>
+
+#### For Next.js
+
+1. Install `sass`:
+
+```bash
+pnpm add -D sass
+```
+
+2. Import the SCSS partials in your main stylesheet:
+
+**File:** `global.css`
+
+```css
+@import 'path-to/styles/_variables.scss';
+@import 'path-to/styles/_keyframe-animations.scss';
+```
+
+#### For Vite + React
+
+1. Install `sass-embedded`:
+
+```bash
+pnpm add -D sass-embedded
+```
+
+2. Import the SCSS partials in your main stylesheet:
+
+**File:** `src/index.css`
+
+```css
+@import 'path-to/styles/_variables.scss';
+@import 'path-to/styles/_keyframe-animations.scss';
+```
+
+Make sure the SCSS files are in the correct path so the styles load properly in your project.
+
+Let me know if you'd like to include support instructions for other tools or frameworks.
```

Comment:
How can a customer let us know that they need an installation guide for another framework? If you want the customer to contact the Tiptap team, you have to provide it a way to do so.

I'd suggest telling customers to reach out to humans@tiptap.dev or, even better, to open an issue https://github.com/ueberdosis/tiptap-docs/issues 

@ThisDavidRichard am I right with this?